README.md

```diff
@@ -306,6 +306,7 @@ $ kubectl delete -f deploy/operator.yaml
 $ kubectl delete -f deploy/role_binding.yaml
 $ kubectl delete -f deploy/role.yaml
 $ kubectl delete -f deploy/service_account.yaml
+$ kubectl delete -f deploy/crds/cache_v1alpha1_memcached_crd.yaml
```

Comment:
One non-blocking nit: move this to line 305

Reply:
Done